contextbasedrestrictionsv1/context_based_restrictions_v1.go

```diff
@@ -50,15 +50,15 @@ const DefaultServiceURL = "https://cbr.cloud.ibm.com"
 // DefaultServiceName is the default key used to find external configuration information.
 const DefaultServiceName = "context_based_restrictions"
 
-// Options : Service options
-type Options struct {
+// ContextBasedRestrictionsV1Options : Service options
+type ContextBasedRestrictionsV1Options struct {
```

Comment:
changed back

Reply:
@padamstx I agree with this statement. However, it seems the first person to publish already started making changes to the auto-generated names. My concern is that if we allow this longer name to stay that we'll have to do a major version bump since this would be a breaking change to the struct name. If you're ok with this (or have some other strategy to handle this), I'm ok with sticking with the auto-generated name.